### PR TITLE
CSV for camera info and new 3 class model

### PR DESCRIPTION
**Pull request by acgold**
- Camera info and URLs are now loaded using a .csv file that is easy to edit. Includes column "use" that can be set to "T" or "F" so only some rows are used
- The model in use is now a model that classifies image into 3 classes (flooding, not sure, and no flooding) and returns probabilities for each class. The highest probability class is displayed as the picture label and the probability is displayed beneath each picture.
- Edited code to fix tide display bug #13

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -48,20 +48,12 @@ tmp_dir <- tempdir()
 
 #------- camera list --------------------
 
-# This could be a .csv so it's easier to edit. Column names = "camera_list", "url", "lat", "long"
-cam_list <- c("Mirlo", "NorthDock", "SouthDock", "SouthOcracoke")
-cam_url_list <- c('https://tims.ncdot.gov/TIMS/cameras/viewimage.ashx?id=NC12_MirloBeach.jpg',
-                  'https://tims.ncdot.gov/TIMS/Cameras/viewimage.ashx?id=Hatteras_Inlet_North_Dock.jpg',
-                  'https://tims.ncdot.gov/TIMS/Cameras/viewimage.ashx?id=Hatteras_Inlet_South_Dock.jpg',
-                  'https://tims.ncdot.gov/tims/cameras/viewimage.ashx?id=Ocracoke_South.jpg')
+# Lat and Long aren't currently in use but exist in the csv for later mapping
```
Comment (anardek) [inline app.R:51]: That's awesome. Will be useful.

File: app.R [hunk 2]
```diff
@@ -131,7 +107,7 @@ get_tides <- function(location) {
   
   df <- noaaoceans::query_coops_data(
     station_id = station_id,
-    start_date = format(lubridate::with_tz(Sys.time(), "America/New_York") %>% as.Date(), "%Y%m%d"),
+    start_date = format(lubridate::with_tz(Sys.time(), "America/New_York") %>% as.Date()-1, "%Y%m%d"),
```
Comment (anardek) [inline app.R:110]: so the fix is to pull over three days? smart

**Review by anardek — COMMENTED** [review]
Looks good to me (didn't check that it runs...but I haven't been able to get Rstudio to work yet).

File: app.R (hunk 2 above)
Comment (acgold) [reply app.R:110]: We'll see if it works correctly! Will have to check it later tonight when the next time is tomorrow

**Pull request merged**